app/controllers/api/v1/daily_sleeping_times_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Api::V1::DailySleepingTimesController < ApplicationController
+  permited_params only: %i[create update], params: %i[id date bed_time wake_up_time]
+
+  def index
+    @pagy, @daily_sleeping_times = pagy(current_user.daily_sleeping_times,
+                                        items: Settings.pagination.sleeping_time_items)
+    res @daily_sleeping_times, meta: pagination_dict(@pagy)
+  end
+
+  def create
+    res current_user.daily_sleeping_times.create!(permited_params), status: :created
+  rescue StandardError => e
```

Comment:
You can use the normal rails flow here:
```
def create
  sleeping_time = current_user.daily_sleeping_times.build(sleeping_time_params)
  
  if sleeping_time.save
    render_resource(sleeping_time) # fake method
  else
    render_error(sleeping_time) # fake method
  end
end
```

Reply:
Thank you for your comment <3, I want to rescue all error in the rails application! It looks a bit stupid!
This is my mistake so I have refactored this pull request:
https://github.com/vantrongtran/good-night/pull/8
I don't want to handle all success and error cases in many methods with each controller.